bundle/edu.gemini.ags.client.impl/src/main/scala/edu/gemini/ags/client/impl/AgsHttpClient.scala

```diff
@@ -1,16 +1,21 @@
 package edu.gemini.ags.client.impl
 
-import edu.gemini.model.p1.immutable.Observation
 import edu.gemini.ags.client.api._
+import edu.gemini.model.p1.immutable.Observation
+import edu.gemini.shared.util.LruCache
```

Comment:
There's so little to configuring an LRU cache that I think I'd be inclined to just instantiate it directly since the alternative is adding another jar file to the distribution.